Tests/IntegrationTests/Roslyn/RoslynGenTests.cs

```diff
@@ -90,27 +94,33 @@ private IPackageManager CreatePackageManager(string rootDir)
             return mgr;
         }
 
-        private void AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath)
+        private IPackage AddPackage(IPackageManager manager, string id, string version, string payloadAssemblyFilePath)
         {
             PackageBuilder builder = new PackageBuilder();
             builder.Id = id;
+            builder.Title = "dummy title";
             builder.Version = new SemanticVersion(version);
             builder.Description = "dummy description";
             builder.Authors.Add("dummy author");
+            builder.Owners.Add("dummy owner");
+            builder.ProjectUrl = new System.Uri("http://dummyurl/");
 
             PhysicalPackageFile file = new PhysicalPackageFile();
             file.SourcePath = payloadAssemblyFilePath;
             file.TargetPath = "analyzers/" + Path.GetFileName(payloadAssemblyFilePath);
             builder.Files.Add(file);
 
+            ZipPackage pkg;
```

Comment:
You don't need to define pkg outside the using block, you can return after InstallPackage. the stream is guaranteed to be disposed.